src/server.h

```diff
@@ -1142,6 +1142,9 @@ typedef struct client {
     int resp;               /* RESP protocol version. Can be 2 or 3. */
     redisDb *db;            /* Pointer to currently SELECTed DB. */
     robj *name;             /* As set by CLIENT SETNAME. */
+    robj *lib_name;         /* As set by CLIENT SETINFO. */
+    robj *lib_ver;          /* As set by CLIENT SETINFO. */
```

Comment:
```suggestion
    robj *lib_name;         /* The client library name as set by CLIENT SETINFO. */
    robj *lib_ver;          /* The client library version as set by CLIENT SETINFO. */
```
Otherwise these comments are pretty vague.